internal/storage/metricstore/elasticsearch/reader.go

```diff
@@ -107,16 +99,24 @@ func (r MetricsReader) GetCallRates(ctx context.Context, params *metricstore.Cal
 		metricDesc:          "calls/sec, grouped by service",
 		boolQuery:           r.buildQuery(params.BaseQueryParameters, timeRange),
 		aggQuery:            r.buildCallRateAggregations(params.BaseQueryParameters, timeRange),
```

Comment:
building a query, not aggregation
```suggestion
		aggQuery:            r.buildCallRateAggQuery(params.BaseQueryParameters, timeRange),
```

Reply:
Solved at [36c1826](https://github.com/jaegertracing/jaeger/pull/7295/commits/36c1826c1263b011e195e2e896d5806001d4433f)